src/shared/utilities/messages.ts

```diff
@@ -81,3 +82,47 @@ export function showOutputMessage(message: string, outputChannel: vscode.OutputC
     outputChannel.appendLine(message)
     getLogger().info(message)
 }
+
+interface ProgressReport {
+    message?: string
+    increment?: number
+}
```

Comment:
don't need a named type for this. But if we are going to add it, it should be used everywhere we use vscode.Progress

Reply:
For some reason VS Code doesn't export their own ProgressReport interface :(
I'll remove it to prevent confusion.